### DISCUSSION TITLE
Who's using OpenTelemetry Ruby

### DISCUSSION THREAD

**Discussion by fhwang** [open-telemetry/opentelemetry-ruby#629] [Ideas]
Would it be worth it to collect and maintain a list of companies that are using OTel-Ruby in production? Might make it easier for others to take the leap.

If so, I'm happy to do the legwork to get that started.

**Comment by fbogsany** [comment]
Shopify is using OpenTelemetry Ruby in production.

**Comment by treyhyde** [comment]
Fulcrumapp.com is using OTEL Ruby in production among other OTEL enabled services (Istio, Knative, NodeJS, Go) .
